styles/style.css

```diff
@@ -1,10 +1,217 @@
+/*inserted by Prabesh ------------------------------------------------------------- 
+*Bikes-for-refugees
+
+*/
+
+/* temp styling */
+
+
+
+/* main styling for code your futrue week 1 tutorial by PRABESH recreation of the page without using bootstrap*/
+header {
+    max-width:85%;
```

Comment:
It's normal to leave a single space between css properties and the value, eg 'max-width: 85%' rather than 'max-width:85%'